string source = rogr
string target = liftdrivecamera
time timestamp

